pkg/image/view.go

```diff
@@ -339,13 +389,11 @@ const ErrCoordOutOfRange log.ConstErr = "coordinates out of range"
 
 // SelectPixel adds the given x, y pixel to the
 func (iv *View) SelectPixel(p sdl.Point) error {
-	if iv.selLayer == nil {
-		return nil
-	}
-	if !ui.InBounds(iv.selLayer.area, p) {
-		return nil
+	if iv.selLayer != nil {
+		p.X -= iv.selLayer.area.X
+		p.Y -= iv.selLayer.area.Y
+		return iv.selLayer.SelectTexel(p)
```

Comment:
It makes more sense for this condition to be inversed; by checking the unordinary case, where it's `nil`, we can keep the main logic as far left as possible.